Core/include/Acts/Material/ProtoVolumeMaterial.hpp

```diff
@@ -61,7 +61,7 @@ class ProtoVolumeMaterial : public IVolumeMaterial {
   /// Output Method for std::ostream
   ///
   /// @param sl The outoput stream
-  std::ostream& toStream(std::ostream& sl) const;
+  std::ostream& toStream(std::ostream& sl) const final;
```

Comment:
I would personally default to `override` and only use `final` if there's a reason why it should be. But I'll leave that up to you.

Reply:
No strong opinion.

Reply:
I would prefer to leave this as `final`. This is not a class intended to be inherited from and it prevents misuse. If it needs to in the future, it can be easily change.